### PR TITLE
Login form

### PR DESCRIPTION
**Pull request by valentin519**
Creation of login form

### CODE REVIEW

**Comment by rocambille** [comment]
there are conflicts to resolve in App.css :/

File: studips/src/component/LoginForm.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    }
+    submitForm(e){
+        e.preventDefault();
+    };
+    onChange(e){
+        this.setState({
+            [e.target.name] : e.target.value,
+        });
+    }
+    postForm (){
```
Comment (rocambille) [inline studips/src/component/LoginForm.js]: why not merge submitForm and postForm?

File: studips/src/component/LoginForm.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    secondPageAppear(){
+        document.getElementById('firstPageInscription').style.display = 'none';
+        document.getElementById('secondPageInscription').style.display = 'block'; 
+        document.getElementById('return').style.display = 'block';  
+    }
+    returnFonction(){
+        document.getElementById('firstPageInscription').style.display = 'block';
+        document.getElementById('secondPageInscription').style.display = 'none'; 
+        document.getElementById('return').style.display = 'none';  
+    }
```
Comment (rocambille) [inline studips/src/component/LoginForm.js]: `document.getElementById` is not the react way: use conditional rendering instead :)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: studips/src/component/LoginForm.js [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+    postForm (e){
+        e.preventDefault();
+        const config = {
+            method : "POST",
+            headers : {
+                "Content-type" : "application/json",
+            },
+            body: JSON.stringify(this.state),
+        };
+    const url = "http://localhost8000/user";
```
Comment (rocambille) [inline studips/src/component/LoginForm.js]: following REST conventions, should be http://localhost8000/users with an **s**

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
conflicts to resolve in App.js ;)

File: studips/src/component/LoginForm.js [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+            confirmedPassword: '',
+            email : '',
+            birthday: '',
+            hobbies : '',
+            study : '',
+            bio : '',
+            city : '',
+            nextPage : true,
+        };
+        this.submitForm = this.submitForm.bind(this);
```
Comment (ghost) [inline studips/src/component/LoginForm.js]: submitForm is no where to be found, outdated line

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request merged**